vru_use_case_scripts/README.md

```diff
@@ -142,3 +141,28 @@ Plot the data:
 ### Example output
 
 ![](docs/plot_time_to_collision_example.png)
+
+## `monitor_time_sync_through_logs`
+
+This script monitors real-time whether if all components are stepping in synchronization ensuring their simulation time synchronization.
+> [!NOTE]
+> This script relies on python libraries watchdog and docker
+
+> [!NOTE]
+> Graph is not automatically saved, please be sure to save it before closing if needed
+
+After starting cdasim through carma start all, give few seconds before running:
+```console
+./monitor_time_sync_through_logs
+```
+
+### Example output
+Red line shows how long in system wall time it took for MOSAIC to step to the next simulation time step.
+Blue lines show how long it took ofr each tool to be called by MOSAIC to step to the next simulation time step.
+We should expect red lines under the blue so that all tools are moving forward at the same time.
```

Comment:
Why? I do not understand this logic. You are showing a time difference for each simulation step and then a time different for each tools simulation time steps. Why should one alway be larger than another and what does this prove about time being synchronized. How are you confirm that before CDASim moved from 1900 to 2000 all simulators and other tools were at 1900